lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -107,196 +107,201 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
+                      </button>
+                      <div class="actions-dropdown" id="actions-dropdown-${dashboard_index}" tabindex="-1">
+                        <ul class="actions-dropdown-list" id="actions-dropdown-list-${dashboard_index}" aria-label="${_('Available Actions')}" role="menu">
+                          % if can_unenroll:
+                            <li class="actions-item" id="actions-item-unenroll-${dashboard_index}" role="menuitem">
+                              <% course_refund_url = reverse('course_run_refund_status', args=[six.text_type(course_overview.id)]) %>
+                                  <a href="#unenroll-modal" class="action action-unenroll" rel="leanModal"
+                                    data-course-id="${course_overview.id}"
+                                    data-course-number="${course_overview.number}"
+                                    data-course-name="${course_overview.display_name_with_default}"
+                                    data-dashboard-index="${dashboard_index}"
+                                    data-course-refund-url="${course_refund_url}"
+                                    data-course-is-paid-course="${is_paid_course}"
+                                    data-course-cert-name-long="${cert_name_long}"
+                                    data-course-enrollment-mode="${enrollment.mode}">
+                                    ${_('Unenroll')}
+                                  </a>
+                            </li>
+                        % elif partner_managed_enrollment:
+                          <li class="actions-item" id="actions-item-enrolled-by-partner-${dashboard_index}" role="menuitem">
+                            <a class="action action-message action-unenroll-managed-enrollment">
+                              ${_('You are enrolled by your institution and you should reach out to your institution to drop this course.')}
+                            </a>
+                          </li>
                         % endif
-                      </li>
-                    </ul>
-                  </div>
-                </div>
-            % endif
+                          <li class="actions-item" id="actions-item-email-settings-${dashboard_index}" role="menuitem">
+                            % if show_email_settings:
+                                <a href="#email-settings-modal" class="action action-email-settings" rel="leanModal" data-course-id="${course_overview.id}" data-course-number="${course_overview.number}" data-dashboard-index="${dashboard_index}" data-optout="${course_overview.id in course_optouts}">${_('Email Settings')}</a>
+                            % endif
+                          </li>
+                        </ul>
+                      </div>
+                    </div>
+                % endif
+              </div>
+            </div>
+          </div>
+
+          <div class="continue-button">
```

Comment:
The `continue-button` div is also net new and wraps around the Resume Course/View Course etc. buttons. It's moved to the bottom of the section so we can expand it to fill the width of the card in the mobile view.